_sass/pages/index.scss

```
@@ -656,6 +656,7 @@ section {
     position: relative;
     background: linear-gradient(0deg, $dark 0%, darken($dark, 10) 100%);
     padding: 0;
+    padding-top: 90px;
```

```suggestion
    padding: 90px 0 0 0;
```

May safe some bytes ;-)